projects/app/src/dictionary/dictionary.ts

```
@@ -708,20 +732,45 @@ export function glossOrThrow(
+
+export const allOneSyllablePronunciationsForHanzi = memoize1(
+  async function allOneSyllablePronunciationsForHanzi(
+    hanzi: HanziText,
+  ): Promise<Set<PinyinSyllable>> {
+    const hanziWordMeanings = await lookupHanzi(hanzi);
+    const pronunciations = new Set<PinyinSyllable>();
+
+    for (const [, meaning] of hanziWordMeanings) {
+      for (const pronunciation of meaning.pinyin ?? emptyArray) {
+        if (typeof pronunciation === `string`) {
+          pronunciations.add(pronunciation);
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)